framework/test/unit/modules/reward/endpoint.spec.ts

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+import { Logger } from '../../../../src/logger';
+import { RewardModule } from '../../../../src/modules/reward';
+import { fakeLogger } from '../../../utils/node';
+
+describe('RewardModule', () => {
```

Comment:
```suggestion
describe('RewardModuleEndpoint', () => {
```

`reward_module.spec.ts` also have the same test suite name, also maybe we don't need another describe inside as `endpoint`

Reply:
you are correct, there is a redundancy

Reply:
done